src/internal/model/model.go

```diff
@@ -11,6 +11,11 @@ type (
 	Schema   int
 )
 
+// Schema constants denote the type of model stored. The integer values of the
+// constants can be changed without issue, but the string values should remain
+// the same. If the string values are changed, additional code will be needed to
+// transform from the old value to the new value.
+//
```

Comment:
Seems like we could just use string consts instead of an iota.

Reply:
eh, we could, but not really high on the list of things to do at the moment. Kind of sixes anyway. If golang had builtin support for transforming consts/enums to strings (apart from stringer) and back again then it would make more sense to use consts